src/targets/gpu/fuse_mlir.cpp

```diff
@@ -61,12 +61,19 @@ bool mlir_enabled()
 #endif
 }
 
+const std::vector<std::string>& get_default_options()
+{
+    static std::vector<std::string> default_options{"dot", "convolution", "fused"};
+    return default_options;
+}
+
 static bool is_requested(std::string_view option, bool fallback = false)
 {
     auto string_value = string_value_of(MIGRAPHX_MLIR_USE_SPECIFIC_OPS{}, "");
-    if(string_value.empty())
+    if(string_value.empty() and get_default_options().empty())
         return fallback;
-    const auto options = split_string(string_value, ',');
+    auto options = split_string(string_value, ',');
+    options.insert(options.begin(), get_default_options().begin(), get_default_options().end());
```

Comment:
This should not be concatenating default options with user provided ones as this will mandate "dot", "convolution", "fused" always. I think what you want it use is either a non-empty MIGRAPHX_MLIR_USE_SPECIFIC_OPS{} *or* get_default_options().

Reply:
Missed your comment until now. Just updated to only use get_default_options when MIGRAPHX_MLIR_USE_SPECIFIC_OPS is empty.